src/win32/win_utils.c

```diff
@@ -101,10 +101,6 @@ int local_start()
         merror_exit("WSAStartup() failed");
     }
 
-    /* Initialize error logging for shared modulesd */
-    dbsync_initialize(loggingErrorFunction);
```

Comment:
These calls were moved because they force the linkage of dbsync/rsync and it's still too soon for them to be used anyway.